nsv13/code/modules/munitions/ship_weapons/_ship_weapon.dm

```diff
@@ -344,7 +340,9 @@
  * Transitions from STATE_LOADED to STATE_FED
  */
 /obj/machinery/ship_weapon/proc/feed()
+	set waitfor = FALSE
```

Comment:
Setting waitfor=FALSE here is causing weapons which automatically chamber to fail, because chamber gets tried before this is actually done